src/main/webapp/WEB-INF/web.xml

```diff
@@ -6,94 +6,10 @@
   xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
   version="2.5"
 >
-  <filter>
```

Comment:
Filter instantiation moved to PortalServletModule so dependencies can be injected into these.